wiki/Ranking_Criteria/en.md

```diff
@@ -33,104 +33,80 @@ For the record, these include the recent changes made [from the discussion forum
-- **Every difficulty of a beatmapset must have a minimum drain time of 30 seconds.**
-- **If the drain time of each difficulty is...**
-  - **...lower than 3:30, the lowest difficulty of each included game mode cannot be harder than a Normal.**
-  - **...between 3:30 and 4:15, the lowest difficulty of each included game mode cannot be harder than a Hard.**
-  - **...between 4:15 and 5:00, the lowest difficulty of each included game mode cannot be harder than an Insane.**
-  - **...anything higher, the beatmapset is exempt from reasonable spread rules.**
-  - **Difficulties lower than the highest difficulty can use their play time as a metric instead of drain time.** This does not apply to difficulties with less than 30 seconds of drain time. Judgement regarding the suitability of gameplay elements used for any lowest difficulty Hard and Insane difficulties is up to the Beatmap Nominators and Nomination Assessment Team members for the corresponding game modes.
-- **Difficulty names in a set must be clearly progressive and accurately indicating of their respective difficulties, excluding:**
+- **Every difficulty of a beatmapset must have a minimum [drain time](/wiki/Gameplay/Drain_time) of 30 seconds.**
+- **If the [drain time](/wiki/Gameplay/Drain_time) of each difficulty is...**
+  - **...lower than 3:30**, the lowest difficulty of each included game mode cannot be harder than a Normal.
+  - **...between 3:30 and 4:15**, the lowest difficulty of each included game mode cannot be harder than a Hard.
+  - **...between 4:15 and 5:00**, the lowest difficulty of each included game mode cannot be harder than an Insane.
+  - **Difficulties below the highest difficulty can combine break times with drain time to meet the above thresholds.** This does not apply to difficulties with less than 30 seconds of drain time. Judgement regarding the suitability of gameplay elements used for any lowest difficulty Hard and Insane difficulties is up to the [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators/) and [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) members for the corresponding game modes.
+- **Difficulty names in a beatmapset must be clearly progressive and accurately indicating of their respective difficulties, excluding:**
   - The highest difficulty.
   - The highest difficulties with a similar level of difficulty, applying only to Insane and Extra difficulties (e.g. the Insane difficulties of a ENHIIII set or the Extra difficulties of a ENHIIXXX set).
+- **All game modes within a beatmapset must form a spread starting from the lowest difficulty level dictated by the song's [drain time](/wiki/Gameplay/Drain_time),** excluding single-difficulty beatmapsets. Each difficulty must comply with its respective mode's difficulty-specific Ranking Criteria, the spread cannot skip any difficulty levels, and there cannot be any drastically large difficulty gaps between any two difficulties.
```

Comment:
"excluding single-difficulty beatmapsets" is useless, a single diff set at appropriate difficulty does comply with the spread rule